### DISCUSSION TITLE
Traducción - Bad & Good

### DISCUSSION THREAD

**Discussion by zavbala** [reactjs/es.reactjs.org#683] [Q&A] [answered]
Estoy a cargo de traducir esta página [unknown-prop](https://github.com/reactjs/es.react.dev/blob/32f55d0edf8730f798428406b9801022b14a4003/src/content/warnings/unknown-prop.md)

Debajo del contenido están las palabras: `bad` & `good`

¿Cuál es la traducción correcta para estas?
¿Alguna sugerencia?

Es decir, lo natural es:

1. Bad = Incorrecto
2. Good = Correcto

¿Estamos de acuerdo?

No logré encontrar otro recurso que esté usando estas palabras.

**Comment by carburo** [comment, marked as answer]
Sí, podemos avanzar con esa traducción. No creo que hasta ahora haya aparecido en otra página una estructura similar.
